viewing_party/party.py

```diff
@@ -0,0 +1,227 @@
+# ************   Wave 1   ************
+def create_movie(title, genre, rating):
+    # return a dictionary if all parameters are Truthy.
+    # return None if any parameter is Falsy
+
+    if title == None or genre == None or rating == None:
+        return None
+
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    return new_movie
```

Comment:
👍 Good work! A few tips on refactoring:

1) The more "pythonic" way to check for truthiness/falsiness is `if not title`

2) We can re-create the dictionary in the return statement without sacrificing readability. 

```suggestion
if not title or not genre or not rating:
        return None

    return {
    "title": title, 
    "genre": genre,
        "rating": rating,
    }
```